src/chrome/content/rules/Bilibili.com.xml

```diff
@@ -1,41 +1,122 @@
+		<test url="http://api.bilibili.com/x/reply?jsonp=jsonp&#x26;oid=191437&#x26;type=3" />
 		<test url="http://api.bilibili.com/x/web-show/res/loc?pf=0&#x26;id=142" />
-	<target host="bml.bilibili.com" />
-		<test url="http://bml.bilibili.com/bml_intro?bid=13" />
+
+	<target host="comment.bilibili.com" />
+		<exclusion pattern="^http://comment\.bilibili\.com/cloud/filter/" />
+		<test url="http://comment.bilibili.com/cloud/filter/5000000.json" />
+		<test url="http://comment.bilibili.com/11998315.xml" />
+
 	<target host="data.bilibili.com" />
 		<exclusion pattern="^http://data\.bilibili\.com/$" />
+		<exclusion pattern="^http://data\.bilibili\.com/e/p" />
+		<exclusion pattern="^http://data\.bilibili\.com/v/web/" />
 		<test url="http://data.bilibili.com/a/access.js" />
-	<target host="passport.bilibili.com" />
+		<test url="http://data.bilibili.com/e/p" />
+		<test url="http://data.bilibili.com/v/web/live_web_statistics?mid=698051" />
+		<test url="http://data.bilibili.com/v/web/web_page_view?mid=698051" />
+
+	<!--	MCB:	-->
+	<target host="app.bilibili.com" />
+		<exclusion pattern="^http://app\.bilibili\.com/(?!images/)" />
+		<test url="http://app.bilibili.com/images/android.html" />
+
+		<test url="http://app.bilibili.com/mobile/android.html" />
+
+	<target host="bangumi.bilibili.com" />
+		<exclusion pattern="^http://bangumi\.bilibili\.com/moe/2016/" />
+		<test url="http://bangumi.bilibili.com/sponsor/rankweb/get_sponsor_week_list" />
+		<test url="http://bangumi.bilibili.com/web_api/user_area" />
+		<test url="http://bangumi.bilibili.com/22/" />
+		<test url="http://bangumi.bilibili.com/anime/1111" />
+
+		<test url="http://bangumi.bilibili.com/moe/2016/jp/index" />
+		<test url="http://bangumi.bilibili.com/moe/2016/jp/character/list" />
+
+	<target host="bml.bilibili.com" />
+		<exclusion pattern="^http://bml\.bilibili\.com/(?!bml_intro)" />
+		<test url="http://bml.bilibili.com/bml_intro" />
+		<test url="http://bml.bilibili.com/bml_intro?bid=13" />
+
+		<test url="http://bml.bilibili.com/index2016/" />
+		<test url="http://bml.bilibili.com/micro2015/" />
+
+	<target host="h.bilibili.com" />
+		<exclusion pattern="^http://h\.bilibili\.com/(?!css/|js/|images/|img/)" />
+		<test url="http://h.bilibili.com/css/draw_yoo.css" />
+		<test url="http://h.bilibili.com/js/picwall.js" />
+		<test url="http://h.bilibili.com/images/ra.js" />
+		<test url="http://h.bilibili.com/img/Icons2.png" />
+
+		<test url="http://h.bilibili.com/nav.js" />
+		<test url="http://h.bilibili.com/dy190003" />
+		<test url="http://h.bilibili.com/dy191437" />
+		<test url="http://h.bilibili.com/index/pic_banner.json" />
+		<test url="http://h.bilibili.com/login" />
+		<test url="http://h.bilibili.com/member?mod=addbanner&#x26;id=41" />
+
+	<!--	Directly:	-->
+	<target host="account.bilibili.com" />
+		<test url="http://account.bilibili.com/site/nameplate.html" />
+	<target host="big.bilibili.com" />
```

Comment:
Mixed Content: The page at 'https://big.bilibili.com/site/big.html' was loaded over HTTPS, but requested an insecure resource 'http://vip.bilibili.com/site/vip-exchange-plugin.html?'. This request has been blocked; the content must be served over HTTPS.